dashboard/test/models/pd/scholarship_info_test.rb

```diff
@@ -42,4 +42,9 @@ class Pd::ScholarshipInfoTest < ActiveSupport::TestCase
     scholarship_info.reload
     assert_equal Pd::ScholarshipInfoConstants::YES_OTHER, scholarship_info.scholarship_status
   end
+
+  test 'cannot create CSF scholarship info with CSP-specific scholarship status' do
```

Comment:
you might consider adding a test to verify that you can create a CSP scholarship with the course-specific status (since this is new)

Reply:
👍 added!